src/FSharp.Data.GraphQL.Samples.GiraffeServer/Schema.fs

```diff
@@ -142,14 +142,14 @@ module Schema =
             isTypeOf = (fun o -> o :? Human),
             fieldsFn = fun () -> 
             [
-                Define.Field("id", String, "The id of the human.", fun _ h -> h.Id)
-                Define.Field("name", Nullable String, "The name of the human.", fun _ h -> h.Name)
+                Define.Field("id", String, "The id of the human.", fun _ (h : Human) -> h.Id)
```

Comment:
For some reason, implementing the possibility of weighted fields as an argument to the Define.Field makes some of those functions to not recognize the correct argument type. I need to work on that somehow.